lib/nerves/utils/http_client.ex

```diff
@@ -76,6 +81,14 @@ defmodule Nerves.Utils.HTTPClient do
     {:noreply, %{s | filename: "", content_length: 0, buffer: "", buffer_size: 0, url: nil}}
   end
 
+  def handle_info({:http, {_ref, {{_, status_code, _}, headers, _body}}}, s) do
```

Comment:
This is how Hex handles redirects: https://github.com/hexpm/hex/blob/master/lib/hex/http.ex#L60-L87